accord-core/src/main/java/accord/local/Bootstrap.java

```diff
@@ -124,8 +124,9 @@ void start(SafeCommandStore safeStore0)
 
             if (!node.topology().hasEpoch(globalSyncId.epoch()))
             {
-                node.topology().awaitEpoch(globalSyncId.epoch())
-                    .addCallback(() -> store.execute(empty(), this::start).begin(node.agent())).begin(node.agent());
+                // Ignore timeouts fetching the epoch, always keep trying to bootstrap
+                // TODO (review): begin wasn't being called so this leaked, but what to do with the error?
```

Comment:
why do you say `begiin` wasn't called?  There are 2 `begin` calls, one for `execute` and one for `addCallback`

Reply:
Comment is out of date, I fixed this earlier in https://github.com/apache/cassandra-accord/commit/37c957c719491634f081b39900ebf708079ef3ee